CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* 3.6.0 (not released yet)
+
+ * Add support for PHP 8.2
```

Comment:
Since "adding support" did not result in any code changes, I'm not convinced this requires a new minor release or even a changelog entry, tbh.

Reply:
Reverted